lib/rules/jsx-handler-names.js

```diff
@@ -24,23 +24,42 @@ module.exports = {
       type: 'object',
       properties: {
         eventHandlerPrefix: {
-          type: 'string'
+          type: ['string', 'boolean']
         },
         eventHandlerPropPrefix: {
-          type: 'string'
+          type: ['string', 'boolean']
         }
       },
       additionalProperties: false
     }]
   },
 
   create(context) {
+    function isPrefixDisabled(prefix) {
+      return (
+        typeof prefix === 'boolean' &&
```

Comment:
this line is redundant with the one after it
```suggestion
```

Reply:
Done